supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -297,18 +297,16 @@ Avoid using the name "OpenShift" on its own when referring to something that app
 
 *See also*: xref:container-registry[container registry], xref:red-hat-container-catalog[Red Hat Container Catalog]
 
-// OCP: General; kept as is
-// TODO: This term is outdated anyway and should be removed in a future update
 [discrete]
 [[openshift-master]]
-==== image:images/yes.png[yes] OpenShift master (noun)
-*Description*: Provides a REST endpoint for interacting with the system and manages the state of the system, ensuring that all containers expected to be running are actually running and that other requests such as builds and deployments are serviced. New deployments and configurations are created with the REST API, and the state of the system can be interrogated through this endpoint as well. An OpenShift master comprises the API server, scheduler, and SkyDNS.
+==== image:images/no.png[no] OpenShift master (noun)
+*Description*: In Red Hat OpenShift, do not use the term "OpenShift master". Use "control plane" instead.
```

Comment:
Is it "OpenShift control plane"? Or just "control plane" Just wanted to be sure.

Reply:
Although you could probably talk about it as the "OpenShift control plane" and technically be right, we really don't. We really just say "the control plane". https://docs.openshift.com/container-platform/4.11/architecture/index.html#about-control-planes